pandas/tests/window/test_rolling.py

```diff
@@ -1691,7 +1692,11 @@ def test_rolling_quantile_interpolation_options(quantile, interpolation, data):
     if np.isnan(q1):
         assert np.isnan(q2)
     else:
-        assert q1 == q2
+        if not IS64:
```

Comment:
Since we have the xfail I think you can remove this branch?

Reply:
I think Github smushed the changes confusingly. 

There are actually two separate tests "test_rolling_var_numerical_issues", and "test_rolling_quantile_interpolation_options" being xfailed.